astropy/io/ascii/ipac.py

```diff
@@ -554,7 +554,6 @@ def write(self, table):
         data_str_vals = list(zip(*self.data.str_vals()))
 
         for i, col in enumerate(table.columns.values()):
-            # FIXME: In Python 3.4, use max([], default=0).
```

Comment:
Not sure what this comment is about but since CI is green, obviously nothing needs fixing anymore, @taldcroft or @hamogu ?

Reply:
I think it means one can replace the `if` statement and its clause with
```
col.width = max(len(vals[i]) for vals in data_str_vals, default=0)
```

Reply:
Thanks, @mhvk ! I added a commit with your suggestion and credited you as co-author.